app/controllers/admin/users_controller.rb

```diff
@@ -1,3 +1,44 @@
 class Admin::UsersController < AdminController
-  def index; end
+  before_action :load_user, only: [:destroy]
+
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    @user.role = 1
+    @user.password = Settings.password
+    @user.password_confirmation = Settings.password
+    if @user.save
+      flash[:success] = t(:done)
+      redirect_to admin_users_path
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t("admin.deleted_msg")
+      redirect_to admin_users_path
+    else
+      flash[:danger] = t(:wrong_msg)
+    end
+  end
+
+  private
+
+  def load_user
+    @user = User.find_by(id: params[:id])
+    render plain: "User not found" if @user.nil?
```

Comment:
`return if @user`
`render plain: "User not found"` ri ko khỏe hơn ah em

ngoài ra, message dùng I18n nào

Reply:
fixed